exchanges/cmd/exchanges.txt

```diff
@@ -0,0 +1,89 @@
+package exchanges
```

Comment:
Is that normal? to have a txt files?

Reply:
I've seen a couple modules follow this approach for converting data downloaded from some external source to golang code. This file is the "template" used for the automatic conversion (https://pkg.go.dev/text/template)